data_model.md

```diff
@@ -0,0 +1,397 @@
+[Use Cases](https://solarforecastarbiter.org/usecases/#forecastdef)
+for more detailed explanation of forecast parameters):
+
+-   *Name* - name of the Forecast
+-   *Issue time of day* - The time of day that a forecast run is issued,
+    e.g. 00:30. For forecast runs issued multiple times within one day
+    (e.g. hourly), this specifies the first issue time of day.
+    Additional issue times are uniquely determined by the first issue
+    time and the run length & issue frequency attribute.
+-   *Lead time to start* - The difference between the issue time and the
+    start of the first forecast interval, e.g. 1 hour.
+-   *Interval length* - The length of time that each data point
+    represents, e.g. 5 minutes, 1 hour.
+-   *Run length / issue frequency* - The total length of a single issued
+    forecast run, e.g. 1 hour. To enforce a continuous, non-overlapping
+    sequence, this is equal to the forecast run issue frequency.
+-   *Interval label* - Indicates if a time labels the beginning or the
+    ending of an interval average, or indicates an instantaneous value,
+    e.g. beginning, ending, instant
+-   *Value type* - The type of the data in the forecast, e.g. mean, max,
+    95th percentile.
+-   *Variable* - The variable in the forecast, e.g. power, GHI, DNI. Each
+    variable is associated with a standard unit.
+-   *Site* - The predefined site that the forecast is for, e.g. Power
+    Plant X or Aggregate Y.
+
+Additional parameters, such as model configuration parameters, may be
+specified under the “extra parameters” key.
+
+
+Each Forecast has data associated with it. The format of this data is
+found in the Data Format section below
+
+### Aggregates
+{: .anchor}
+
+Each Aggregate must be associated with one or more Observations. Solar
+Forecast Arbiter will calculate aggregated observation data generated
+from these Observations. Each aggregate will also provide a list of
+Sites associated with the Observations. These Sites will provide
+forecast providers with the modeling parameters for each system in the
+Aggregate. Aggregate forecasts will be associated directly with an
+Aggregate. Forecast providers will be required to submit a forecast of
+the aggregated quantity.
+
+Aggregates have the following required parameters:
+
+-   *Name* - Name of the Aggregate
+-   *Description* - A description of the makeup of the Aggregate, e.g. all
+    utility scale solar in ISO region
+-   *Timezone* - IANA timezone of the site, e.g. Etc/GMT+8
+-   *Observations* - A list of Observations that make up the aggregate
+
+## Variables and Units
+{: .anchor}
+
+Solar Forecast Arbiter will only accept a standard set of variables and
+units. Data not in in the correct units must be converted before it is
+uploaded. The forecast/observation variables and units that are
+permitted are as follows:
+
+-   *surface temperature* - degrees Celsius
+-   *surface wind speed* - meters per second
```

Comment:
ah, I forgot to add that these surface weather variables should start with capital S